package/bin/edges_metrics_helper.py

```diff
@@ -8,6 +8,7 @@
 from splunklib import modularinput as smi
 
 from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
See other comments..

```suggestion
from datetime import datetime, timedelta
```